src/client/containers/PullRequestSummary/ReviewSection/ReviewerList/index.js

```diff
@@ -29,25 +25,104 @@ const getStateClass = (state: string) => {
 
 const getClass = (icon: string, state: string) => `fa ${icon} ${getStateClass(state)}`
 
-const ReviewerList = (props: ReviewerListType) => {
-  if (props.reviewers.length === 0) {
-    return null
-  }
-  const tooltipOverlay = <Tooltip id="tooltip">{props.tooltip}</Tooltip>
-  return (
-    <li>
-      <span>
-        <OverlayTrigger placement="left" overlay={tooltipOverlay}>
-          <i
-            style={{ marginRight: '12px', width: '12px', marginBottom: '6px' }}
-            className={getClass(props.icon, props.approval)}
-          />
-        </OverlayTrigger>
-        {props.reviewers.map(r => r.user.fullName || r.user.username).join(', ')}
-      </span>
-    </li>
+export const getReviews = createSelector(
```

Comment:
and also it seems that we should put the creation of the selectors in ducks/   as we just talk I think it make sense have some clarifying guidelines